p11kit/attribute.go

```diff
@@ -78,8 +78,10 @@ func (o *Object) SetLabel(label string) {
 	})
 }
 
-// SetCertificate set certificate attributes to key object.
-// It will create the link between key and certificate.
+// SetCertificate associates a public or private key with a certificate. This is
+// required for many clients to know which key corresponds to which certificate.
+//
+// This method will return an error if the object isn't a public or private key.
```

Comment:
I can look it up in the spec if you don't know off the top of your head, but intermediary certs would we want them to potentially have the same ID? Or are clients expected to use the issuer in order to build up the chain?

Reply:
Added a TODO for now and opened #26